src/aks-preview/azext_aks_preview/tests/latest/test_helpers.py

```diff
@@ -63,5 +67,22 @@ def test_trim_fqdn_name_trailing_dash(self):
     def test_trim_fqdn_name_not_containing_hcp(self):
         container_name = 'abcdef-dns-ed55ba6d-e48fe2bd-b4bc-4aac-bc23-29bc44154fe1-privatelink-centralus-azmk8s-io'
         expected_container_name = 'abcdef-dns-ed55ba6d-e48fe2bd-b4bc-4aac-bc23-29bc44154fe1-privat'
-        trim_container_name = helpers._trim_fqdn_name_containing_hcp(container_name)
+        trim_container_name = helpers._trim_fqdn_name_containing_hcp(
+            container_name)
         self.assertEqual(expected_container_name, trim_container_name)
+
+
+class TestFuzzyMatch(unittest.TestCase):
+    def setUp(self):
+        self.expected = ['bord', 'birdy', 'fbird', 'bir', 'ird', 'birdwaj']
+
+    def test_fuzzy_match(self):
+        result = helpers._fuzzy_match(
+            "bird", ["plane", "bord", "birdy", "fbird", "bir", "ird", "birdwaj", "bored", "biron", "bead"])
+
+        self.assertCountEqual(result, self.expected)
+        self.assertListEqual(result, self.expected)
+
+
+if __name__ == "__main__":
```

Comment:
Do we need this here all the time or just for local testing? I think it can be removed as I am guessing the the tests are run at the root of the AKS Cli repo (similar to a go test ./...) and in that case I doubt we need the main part

Reply:
Yea, it's to run individual tests locally. Not needed for the main repo, but I figured I'd leave it there in case future engineers want to run a single test.

Reply:
Better to take it out and let them add it. 

I think it could cause issues with other higher level tests trying to import ant run them

Reply:
Are you sure? Because I think the line returns false unless the user is explicitly calling the file. So it shouldn't affect anything if the file is imported.  Other files, such as the `test_loadbalancer.py` also have the line.

Reply:
@gtracer  any update?

Reply:
Talked offline. @gtracer is fine with leaving it. Thanks!